webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersTable/UserCredentialsList.tsx

```diff
@@ -32,11 +36,43 @@ export const UserCredentials = observer<IUserCredentialsProps>(function UserCred
 });
 
 export const UserCredentialsList: PlaceholderComponent<IUserDetailsInfoProps> = observer(function UserCredentialsList({ user }) {
-  return (
+  const styles = useS(style);
+  const translate = useTranslate();
+  const menu = useMenuState({
+    placement: 'top',
+    gutter: 8,
+  });
+
+  const visibleCredentials = user.origins.slice(0, MAX_VISIBLE_CREDENTIALS);
+
+  return styled(BASE_DROPDOWN_STYLES)(
     <Fragment key="user-credentials-list">
-      {user.origins.map(origin => (
+      {visibleCredentials.map(origin => (
         <UserCredentials key={origin.type + origin.subType} origin={origin} />
       ))}
-    </Fragment>
+
+      {user.origins.length > MAX_VISIBLE_CREDENTIALS && (
+        <>
+          <MenuButton {...menu} className={s(styles, { menuButton: true })}>
+            <div className={s(styles, { hasMoreIndicator: true })}>
+              <span>+{user.origins.length - MAX_VISIBLE_CREDENTIALS}</span>
```

Comment:
Not a big fan of "+" for type casting. it is easy to forget or accidentally remove. More safe approach:

`<span>{Number(user.origins.length - MAX_VISIBLE_CREDENTIALS)}</span>`